packages/RT128156/Needed.pm6

```diff
@@ -1 +1,2 @@
 class Needed {};
+#
```

Comment:
Unsure if this was an intentional change. A test in `t/spec/packages/RT128156/Needed.pm6` intentionally writes comment symbols to this file during spectest run and I saw in this past this file left changed if spectest is prematurely aborted.